pages/admin/request/create-renewal.tsx

```diff
@@ -61,18 +76,203 @@ export default function CreateRenewal() {
+          firstName: data.applicant.firstName,
+          lastName: data.applicant.lastName,
+          email: data.applicant.email,
+          phone: data.applicant.phone,
+          addressLine1: data.applicant.addressLine1,
+          addressLine2: data.applicant.addressLine2,
+          city: data.applicant.city,
+          postalCode: data.applicant.postalCode,
+        });
+
+        // set doctorInformation
+        const physician = data.applicant.medicalInformation.physician;
+        setDoctorInformation({
+          phone: physician.phone,
+          addressLine1: physician.addressLine1,
+          addressLine2: physician.addressLine2,
+          city: physician.city,
+          postalCode: physician.postalCode,
+          name: physician.name,
+          mspNumber: physician.mspNumber,
+        });
+
+        // set additionalQuestions
+        if (data.applicant.mostRecentRenewal.renewal) {
+          setAdditionalQuestions({
+            usesAccessibleConvertedVan:
+              data.applicant.mostRecentRenewal.renewal.usesAccessibleConvertedVan,
+            requiresWiderParkingSpace:
+              data.applicant.mostRecentRenewal.renewal.requiresWiderParkingSpace,
+          });
+        } else {
+          setAdditionalQuestions({
+            usesAccessibleConvertedVan: false,
+            requiresWiderParkingSpace: false,
+          });
+        }
+
+        // set paymentDetails
+        //TODO: maybe don't pre-populate payment info
+        const previousApplication = data.applicant.mostRecentRenewal;
+        setPaymentDetails({
+          paymentMethod: previousApplication.paymentMethod,
+          donationAmount: 0,
+          shippingAddressSameAsHomeAddress: previousApplication.shippingAddressSameAsHomeAddress,
+          shippingFullName: previousApplication.shippingFullName,
+          shippingAddressLine1: previousApplication.shippingAddressLine1,
+          shippingAddressLine2: previousApplication.shippingAddressLine2,
+          shippingCity: previousApplication.shippingCity,
+          shippingProvince: previousApplication.shippingProvince,
+          shippingPostalCode: previousApplication.shippingPostalCode,
+          billingAddressSameAsHomeAddress: previousApplication.billingAddressSameAsHomeAddress,
+          billingFullName: previousApplication.billingFullName,
+          billingAddressLine1: previousApplication.billingAddressLine1,
+          billingAddressLine2: previousApplication.billingAddressLine2,
+          billingCity: previousApplication.billingCity,
+          billingProvince: previousApplication.billingProvince,
+          billingPostalCode: previousApplication.billingPostalCode,
+        });
+      },
+    }
+  );
+
+  /**
+   * Set and fetch data about applicant when permit holder is selected
+   */
+  const handleSelectedPermitHolder = async (permitHolder: PermitHolder | undefined) => {
+    setPermitHolderRcdUserID(permitHolder?.rcdUserId || undefined);
```

Comment:
```suggestion
    setPermitHolderRcdUserID(permitHolder?.rcdUserId);
```
`undefined` should be redundant when we have optional chaining (`?`)

Reply:
`permitHolder?.rcdUserId` returns `null` if its undefined, so your suggestion gives this error: `Type 'null' is not assignable to type 'SetStateAction<number | undefined>'.` Going to leave it as is for now.